aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,66 +1,14 @@
 ---
 remote_endpoint: ~
-name: "v1.9"
+name: "v1.10"
 proposals:
-  - name: step_1_increase_max_txn_gas
```

Comment:
we need to keep this step , until exception for framework execution max gas charging is implemented

Reply:
I added the step back. How does it look?